### PR TITLE
HIGH PRIORITY: xcuitestbrowserstack 1.0.7

### PR DESCRIPTION
**Pull request by satheeshkumarf**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [Yes] __I will not move an already shared step version's tag to another commit__
- [Yes] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [Yes ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [Yes ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [Yes ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by satheeshkumarf** [comment]
@here Can we please merge is PR asap, as this is an import one :)

File: steps/xcuitestbrowserstack/1.0.7/step.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    summary: Specifiy path to the file which contains your test and device details.
+      Use .txt file to specify your test name and device details separated by equal
+      to sign.
+    title: Test file path
+  tests_file_path: null
+- browserstack_xcuitest_url: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Build your XCUITests and publish it to Browserstack sever. Specify the
```
Comment (koral--) [inline steps/xcuitestbrowserstack/1.0.7/step.yml]: Looks like a typo: `sever` -> `server`

File: steps/xcuitestbrowserstack/1.0.7/step.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+website: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack
+source_code_url: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack
+support_url: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack/issues
+published_at: 2019-11-21T21:41:06.877327+04:00
+source:
+  git: https://github.com/satheeshkumarf/bitrise-step-executexcuitestonbrowserstack.git
+  commit: 77044ef1626f731f5c2fcc3b9e1c7956a61b4261
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
```
Comment (koral--) [inline steps/xcuitestbrowserstack/1.0.7/step.yml]: Does it make sense to run this step on Linux?

File: steps/xcuitestbrowserstack/1.0.7/step.yml (hunk 1 above)
Comment (satheeshkumarf) [reply steps/xcuitestbrowserstack/1.0.7/step.yml]: Created a new PR https://github.com/bitrise-io/bitrise-steplib/pull/2281/files

@koral--

**Pull request closed without merging**